.github/actions/mamba-env/action.yml

```diff
@@ -0,0 +1,44 @@
+name: "Install environment using Mamba"
+description: "Prepare the environment to run Modin"
+inputs:
+  python-version:
+    description: "Python version to install"
+    default: "3.8"
+  environment-file:
+    description: "Conda environment yml"
+    required: true
+  activate-environment:
+    description: "Conda environment to activate"
+    default: "modin"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Get current week
```

Comment:
In a previous version, we didn't have a staling at all, and (usually) it didn't hurt. 🙃 
The probable reason is that we have so many different cached variants, that GH evicts some of our caches quite frequently.

So I would propose to keep this change as it is now and (maybe) address the eviction later when some statistics on it could be gathered.